poc/nvidia/cleanup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This will completely remove microk8s from pc. We can add small warning in README maybe user was using the microk8s for somthing else not just dss.

Reply:
Good point! Pushed a commit